docs/faq.rst

```diff
@@ -92,3 +92,22 @@ preview" status. However, you can take a look at `this blog post`_ outlining a
 workable "DIY" solution as a stop gap for now.
 
 .. _`this blog post`: https://www.jeffgeerling.com/blog/2019/how-add-integration-tests-ansible-collection-molecule
+
+Does Molecule support monorepos?
+================================
+
+Yes, roles contained in a monorepo with other roles are automatically picked up and the `ANSIBLE_ROLES_PATH` is set accordingly. See `this page`_ for more information
```

Comment:
The '_\`this page\`' link should use an Intersphinx link which will resolve the stable/latest/version blah blah that needs to be handled for Sphinx and ReadTheDocs in the long-term. Please check the contributor section for how to work with Intersphinx.

Reply:
For reference: https://github.com/ansible/molecule/blob/master/CONTRIBUTING.rst#working-with-intersphinx

Reply:
I tried this approach but it fails me. Any ideas?

```
$ python -m sphinx.ext.intersphinx https://molecule.readthedocs.io/en/stable/examples.html
/home/per/git/ansible-installation/.venv/bin/python: Error while finding module specification for 'sphinx.ext.intersphinx' (ModuleNotFoundError: No module named 'sphinx')
```

Reply:
```
$ pip install sphinx
```

?

Reply:
Maybe we can improve https://molecule.readthedocs.io/en/stable/contributing.html#working-with-intersphinx ... I know most don't work with this part of Sphinx and it can be a bit cryptic ...

Reply:
@decentral1se Ah, of course. I think I copied the wrong error message; this was the one I got stuck with:

```
$ python -m sphinx.ext.intersphinx https://molecule.readthedocs.io/en/stable/examples.html
intersphinx inventory 'https://molecule.readthedocs.io/en/stable/examples.html' not readable due to ValueError: ('intersphinx inventory %r not readable due to %s: %s', 'https://molecule.readthedocs.io/en/stable/examples.html', 'ValueError', ValueError(...))
```

But I realize now I should have run `python -m sphinx.ext.intersphinx https://docs.ansible.com/ansible/latest/objects.inv` instead, just like it said. I was presuming I should replace the URL there with my URL...

Anyway, I now got the proper output but I have no idea what to do with it. It also included an error:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/per/.local/lib/python2.7/site-packages/sphinx/ext/intersphinx.py", line 432, in <module>
    inspect_main(argv=sys.argv[1:])  # type: ignore
  File "/home/per/.local/lib/python2.7/site-packages/sphinx/ext/intersphinx.py", line 423, in inspect_main
    print(exc.args[0] % exc.args[1:])
TypeError: not all arguments converted during string formatting
```

Any ideas?

Reply:
Ah, not sure. Let's just leave it. What a pain :sweat: